engine/particle/src/wscript

```diff
@@ -32,11 +32,11 @@ def build(bld):
     is_host = bld.env['PLATFORM'] in ('x86_64-linux', 'x86_64-win32', 'x86_64-darwin')
 
     if is_host:
-        bld.shlib(features = 'cxx ddf',
+        o = bld.shlib(features = 'cxx ddf skip_asan',
                   includes = ['.', '../proto'],
                   target = 'particle_shared',
                   protoc_includes = '../proto',
-                  uselib = 'DDF DLIB PLATFORM_SOCKET PROFILE_NULL',
+                  use = 'DDF_NOASAN DLIB_NOASAN PLATFORM_SOCKET PROFILE_NULL_NOASAN',
```

Comment:
I think it should be possible to run these shared libraries with asan/ubsan when invoked by Python, but I haven't investigated it. Until then, let's keep the extra non-asan libraries for this purpose.